airflow/api_fastapi/core_api/routes/public/backfills.py

```diff
@@ -213,9 +212,21 @@ def create_backfill(
         )
     except DagNoScheduleException:
```

Comment:
I think we can consolidate all the 422 Unprocessable Entity errors here and extract the original error from `e.message`. WDYT?  

```suggestion
     except (
        AlreadyRunningBackfill,
        DagNoScheduleException,
        InvalidReprocessBehavior,
        InvalidBackfillDirection,
    ) as e:
        raise HTTPException(
            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
            detail=e.message,
        )
```

Reply:
Implemented this https://github.com/apache/airflow/pull/45731#discussion_r1924092898